src/posts/flox-smart/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: 'flox: GroupBy, now with smarts!'
+date: '2024-05-31'
+authors:
+  - name: Deepak Cherian
+    github: dcherian
+
+summary: 'flox adds heuristics for automatically choosing an appropriate strategy with dask arrays!'
+---
+
+## TL;DR
+
+`flox>=0.9` adds heuristics for automatically choosing an appropriate strategy with dask arrays! Here I describe how.
```

Comment:
do better.

Reply:
Are you okay @dcherian? 😅

Reply:
it's not a great summary -- "what strategy"? hehe